src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -736,7 +736,7 @@ public void AcceptV6BoundToAnyV6_Success()
         }
 
         [Fact]
-        [ActiveIssue(5832, PlatformID.AnyUnix)]
+        [PlatformSpecific(PlatformID.Windows)] // https://github.com/dotnet/corefx/issues/5832
```

Comment:
These have been running reliably on Windows so far, so I'm reluctant to throw away the coverage provided by these tests.  If we start seeing issues on Windows, we should reconsider, but for now let's keep them.